src/poetry/console/commands/sync.py

```diff
@@ -34,3 +34,6 @@ class SyncCommand(InstallCommand):
 If you want to use Poetry only for dependency management but not for packaging,
 you can set the "package-mode" to false in your pyproject.toml file.
 """
+
+    def handle(self) -> int:
+        return self._handle_install(True)
```

Comment:
```suggestion
        return self._handle_install(with_synchronization=True)
```
A small nitpick, but let's avoid nameless boolean calls.